CHANGELOG.md

```diff
@@ -15,15 +41,24 @@
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>`
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>-2019`
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>-2022`
-
   Starting with this release, the `quay.io/signalfx/splunk-otel-collector:<version>` docker image manifest has been
   updated to support Windows (2019 amd64, 2022 amd64), in addition to Linux (amd64, arm64, ppc64le).
-
```

Comment:
Please re-add this line break.  Otherwise there won't be a break with following line.

Reply:
See the preview (https://github.com/signalfx/splunk-otel-collector/blob/a0744114d52adae7dc93935c20f0b35574de249f/CHANGELOG.md#-deprecations-) for how the doc will be rendered without the empty lines.

Reply:
I have added back.